core/time/timezone/tz_windows.odin

```diff
@@ -0,0 +1,303 @@
+	"Line Islands Standard Time" =      {"+14", "+14"},     // Pacific/Kiritimati
+	"Marquesas Standard Time" =         {"-0930", "-0930"}, // Pacific/Marquesas
+	"Norfolk Standard Time" =           {"+11", "+12"},     // Pacific/Norfolk
+	"West Pacific Standard Time" =      {"+10", "+10"},     // Pacific/Port_Moresby
+	"Tonga Standard Time" =             {"+13", "+13"},     // Pacific/Tongatapu
+}
+
+iana_to_windows_tz :: proc(iana_name: string, allocator := context.allocator) -> (name: string, success: bool) {
+	win_tz_name_buffer := [128]u16{}
+	status : windows.UError 
+
+	iana_name_wstr := windows.utf8_to_wstring(iana_name, allocator)
+	defer free(iana_name_wstr)
+
+	win_tz_name_len := windows.ucal_getWindowsTimeZoneID(iana_name_wstr, -1, raw_data(win_tz_name_buffer[:]), len(win_tz_name_buffer), &status)
+	if status != .U_ZERO_ERROR {
+		return
+	}
+
+	win_tz_name, err := windows.utf16_to_utf8(win_tz_name_buffer[:win_tz_name_len], allocator)
+	if err != nil {
+		return
+	}
+
+	return win_tz_name, true
+}
+
+local_tz_name :: proc(allocator := context.allocator) -> (name: string, success: bool) {
+	iana_name_buffer := [128]u16{}
+	status : windows.UError
+
+	zone_str_len := windows.ucal_getDefaultTimeZone(raw_data(iana_name_buffer[:]), len(iana_name_buffer), &status)
+	if status != .U_ZERO_ERROR {
+		return
+	}
+
+	iana_name, err := windows.utf16_to_utf8(iana_name_buffer[:zone_str_len], allocator)
+	if err != nil {
+		return
+	}
+
+	return iana_name, true
+}
+
+REG_TZI_FORMAT :: struct #packed {
+	bias:     windows.LONG,
+	std_bias: windows.LONG,
+	dst_bias: windows.LONG,
+	std_date: windows.SYSTEMTIME,
+	dst_date: windows.SYSTEMTIME,
+}
+
+generate_records_from_tzi :: proc(tzi: ^REG_TZI_FORMAT, year: int, abbrevs: TZ_Abbrev, shortnames: ^[dynamic]string, records: ^[dynamic]datetime.TZ_Record, allocator := context.allocator) {
+	dst_time, _ := time.components_to_time(
+		year, 
+		tzi.dst_date.month, 
+		tzi.dst_date.day,
+		tzi.dst_date.day,
+		tzi.dst_date.hour,
+		tzi.dst_date.minute,
+		tzi.dst_date.second,
+	)
+	std_time, _ := time.components_to_time(
+		year, 
+		tzi.std_date.month, 
+		tzi.std_date.day,
+		tzi.std_date.day,
+		tzi.std_date.hour,
+		tzi.std_date.minute,
+		tzi.std_date.second,
+	)
+
+	shortnames_idx := len(shortnames)
+	append(shortnames, strings.clone(abbrevs.dst, allocator))
+	append(shortnames, strings.clone(abbrevs.std, allocator))
+
+	append(records, datetime.TZ_Record{
+		time = time.to_unix_seconds(dst_time),
+		utc_offset = -(i64(tzi.bias) + i64(tzi.dst_bias)) * 60,
+		shortname = shortnames[shortnames_idx],
+		dst = true,
+	})
+	append(records, datetime.TZ_Record{
+		time = time.to_unix_seconds(std_time),
+		utc_offset = -(i64(tzi.bias) + i64(tzi.std_bias)) * 60,
+		shortname = shortnames[shortnames_idx + 1],
+		dst = false,
+	})
+	return
+}
+
+load_region :: proc(reg_str: string, allocator := context.allocator) -> (out_reg: ^datetime.TZ_Region, success: bool) {
```

Comment:
Do the `_load_region` conventions so that the documentation generation is better and has a singular place for it. e.g.
```odin
load_region :: proc(reg: string, allocator := context.allocator) -> (out_reg: ^datetime.TZ_Region, ok: bool) {
	return _load_region(reg, allocator)
}
```